### ISSUE TITLE
German translation of Responsive Design with Bootstrap

### ISSUE THREAD

**Issue by raisedadead** [freeCodeCamp/TranslationExpansion#63]
@undefinedsymbol commented on [Sat Feb 20 2016](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/7150)
- [ ]  Use Responsive Design with Bootstrap Fluid Containers Complete
- [ ] Make Images Mobile Responsive Complete
- [ ] Center Text with Bootstrap Complete
- [ ]  Create a Bootstrap Button Complete 
- [ ] Create a Block Element Bootstrap Button Complete
- [ ] Taste the Bootstrap Button Color Rainbow Complete
- [ ]  Call out Optional Actions with Button Info Complete
- [ ] Warn your Users of a Dangerous Action Complete
- [ ] Use the Bootstrap Grid to Put Elements Side By Side Complete
- [ ] Ditch Custom CSS for Bootstrap Complete
- [ ]  Use Spans for Inline Elements Complete
- [ ] Create a Custom Heading Complete
- [ ] Add Font Awesome Icons to our Buttons Complete
- [ ] Add Font Awesome Icons to all of our Buttons Complete
- [ ] Responsively Style Radio Buttons Complete
- [ ] Responsively Style Checkboxes Complete
- [ ] Style Text Inputs as Form Controls Complete
- [ ] Line up Form Elements Responsively with Bootstrap
- [ ]  Create a Bootstrap Headline Complete
- [ ]  House our page within a Bootstrap Container Fluid Div Complete
- [ ] Create a Bootstrap Row Complete
- [ ] Split your Bootstrap Row Complete
- [ ] Create Bootstrap Wells Complete
- [ ] Add Elements within your Bootstrap Wells Complete
- [ ]  Apply the Default Bootstrap Button Style Complete
- [ ] Create a Class to Target with jQuery Selectors Complete
- [ ]  Add ID Attributes to Bootstrap Elements Complete
- [ ]  Label Bootstrap Wells Complete
- [ ] Give Each Element a Unique ID Complete
- [ ]  Label Bootstrap Buttons Complete
- [ ] Use Comments to Clarify Code

---

@5k9m commented on [Wed May 04 2016](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/7150#issuecomment-216756473)

What's the status of this, @undefinedsymbol? Can I help by any chance?

---

@undefinedsymbol commented on [Thu May 05 2016](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/7150#issuecomment-217113783)

This has not been started, yet. You can take this part and start working on it.

---

@5k9m commented on [Fri May 06 2016](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/7150#issuecomment-217383867)

Thanks for the update! I'll finish the Basic Javascript part first, then try my best here ;-)

**Comment by DaveBlack29** [comment]
Hey guys,
as this issue still seems to be open and I didn't find another translation so far, I started working on this.
I'll be doing the translating first and then I'll work my way through the translation guide an ask for help if needed. Is it correct that you don't want the test sequences to be translated?
Thanks for your help. 
Dave

**Comment by raisedadead** [comment]
@DaveBlack29 Thanks a lot for helping us with the translation. 

Feel free to reach out to us for any assistance in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors).

> Is it correct that you don't want the test sequences to be translated?

That is correct. Currently our infrastructure would support only translation of the challenge instructions (description) part.

Thanks again for contributing, and happy translating!

**Comment by DaveBlack29** [comment]
@raisedadead Thanks for your quick reply. I got the translation done by now. But it seems I'm stuck with my next step in the guide. I installed git and GitHub Desktop. Then I made a fork to freeCodeCamp, also made a local copy. I found the Bootstrap file I translated under the given seeds link. But now I can't open it with GitHubDesktop or start working on it. What am I missing? Thanks for your help.
